contrib/examples/actions/cloudslang-basic.yaml

```diff
@@ -0,0 +1,13 @@
+---
+description: Greet StackStorm from cloudslang ;) !
+enabled: true
+entry_point: ""
+name: cloudslang-basic
+pack: examples
+parameters:
+  flow_path:
```

Comment:
how can i get the packs location on file system? our CLI requires to pass an actual file as input, and if I want to pass a flow from the examples pack I need its location.
Or, can i assume my CWD, and work relatively?